static/docs/commands-reference/pull.md

```diff
@@ -84,11 +84,11 @@ reflinks or hardlinks to put it in the workspace without copying. See
   behavior from the `dvc checkout` command because `dvc pull` in effect performs
   a _checkout_ after downloading files.
 
-- `-R`, `--recursive` - `targets` values is expected to be a directory path.
-  Determines the files to download by searching the named directory and its
-  subdirectories for DVC files to download data for. Along with providing a
-  `target`, or `target` along with `--with-deps` it is yet another way to cut
-  the scope of DVC files to download.
+- `-R`, `--recursive` - `targets` value is expected to be a file along with its
+  directory path or a directory. Determines the files to download by searching
+  the named directory and its subdirectories for DVC-files to download data for
+  .Along with providing a `target`, or `target` along with `--with-deps` it is
+  yet another way to cut the scope of DVC-files to download.
```

Comment:
Ideas:
`file along with its directory path` -> "path to a file or directory, relative to the current location (e.g. `data/models/predict.pkl`)"
`Determines the files` -> `Determines the DVC-files`; Remove `for DVC-files to download data for`
`the named directory` -> `the target directory`
`.Along`: That period shouldn't have a space before, but should have one after.
"`target`, or `target` along with `--with-deps` it is yet another" -> "`target` and/or `--with-deps`, its another"

Reply:
p.s. but again I don't even see this `-R` flag in `dvc gc` atm

Reply:
I will make the changes tomorrow. Thanks for reviewing @jorgeorpinel